manager/manager.go

```diff
@@ -65,11 +41,11 @@ type Manager interface {
 	// Get information about all subcontainers of the specified container (includes self).
 	SubcontainersInfo(containerName string, query *info.ContainerInfoRequest) ([]*info.ContainerInfo, error)
 
-	// Gets all the Docker containers. Return is a map from full container name to ContainerInfo.
+	// Gets all the containers. Return is a map from full container name to ContainerInfo.
 	AllDockerContainers(query *info.ContainerInfoRequest) (map[string]info.ContainerInfo, error)
```

Comment:
I think those interfaces should also be renamed, so it is not specific to Docker.